content/project/trademark-sublicense.adoc

```diff
@@ -0,0 +1,291 @@
+---
+layout: simplepage
+title:  Sublicense Agreement
```

Comment:
To be effective, this trademark needs to be at leased signed off by SPI. I am not 100% it has ever happened, but I think we can assume so

Reply:
> To be effective, this trademark needs to be at leased signed off by SPI. I am not 100% it has ever happened, but I think we can assume so

I think that is a safe assumption as well, though I didn't consider that when inserting this page.

I wasn't making the assumption that SPI had approved the draft sublicense.  I think that whether or not they have approved the sublicense, it existed previously on the wiki and was referenced by other pages.  I assume that if it needs correction or removal, it is easier to do that from the jenkins.io GitHub repository than from the wiki page.